File: 03 Berechnungen/Go/Aufgabe2.go
package main

import "fmt"

var a int = 7

func main() {
	fmt.Println("a =", a)

	a++
	fmt.Println("a++; (a = a + 1) a =", a)

	a--
	fmt.Println("a--; (a = a - 1) a =", a)

	a += 6
	fmt.Println("a += 6; (a = a + 6) a =", a)

	a -= 6
	fmt.Println("a -= 6; (a = a - 6) a =", a)

	a *= 6
	fmt.Println("a *= 6; (a = a * 6) a =", a)

	a /= 6 // In Go geht das!
	fmt.Println("a /= 6; (a = a / 6) a =", a)
}

File: 03 Berechnungen/Pascal/Aufgabe2.pas
Program Aufgabe1;

{$codepage utf8}

Var A: integer = 7;

Begin
    WriteLn('A = ', A);

    inc(A);
    WriteLn('inc(A); (A = A + 1) A = ', A);

    dec(A);
    WriteLn('dec(A); (A = A - 1) A = ', A);

    A += 6;
    WriteLn('A += 6; (A = A + 6) A = ', A);

    A -= 6;
    WriteLn('A -= 6; (A = A - 6) A = ', A);

    A *= 6;
    WriteLn('A *= 6; (A = A * 6) A = ', A);

//  A /= 6;
    WriteLn('A /= 6; geht nicht!');
End.